stuffer/s2n_stuffer.c

```diff
@@ -161,7 +162,7 @@ int s2n_stuffer_release_if_empty(struct s2n_stuffer *stuffer)
 
     GUARD(s2n_stuffer_wipe(stuffer));
```

Comment:
Why is this wipe needed, since resize to a smaller size will wipe?

Reply:
I created this issue to track this on a separate/dedicated PR https://github.com/awslabs/s2n/issues/2096.

Reply:
i think we could probably remove it here safely, thoughts?